### PR TITLE
Update prod tests to point to re-exported 0.5 model

### PR DESCRIPTION
**Pull request by reuben**
This is just the checkpoint from 0.5.0 re-exported with the code from 0.6.0-alpha.0 since we changed the inference graph in #2184

### CODE REVIEW

**Comment by reuben** [comment]
Seems like the macOS workers are dead again, as their tests are pending. But otherwise everything looks green.

**Comment by lissyx** [comment]
> Seems like the macOS workers are dead again, as their tests are pending. But otherwise everything looks green.

they look fine to me: https://tools.taskcluster.net/provisioners/deepspeech-provisioner/worker-types/ds-macos-light

**Comment by lissyx** [comment]
> Seems like the macOS workers are dead again, as their tests are pending. But otherwise everything looks green.

Looks like we will need to fix proper in task-dir scoping for NPM, pip and Electron, there was a 10GB folder ~/.npm/ ...

File: native_client/test/concurrent_streams.py [hunk 1]
```diff
@@ -53,13 +53,15 @@ def main():
...
         ds.enableDecoderWithLM(args.alphabet, args.lm, args.trie, LM_ALPHA, LM_BETA)
 
-    with wave.open(args.audio1, 'rb') as fin:
-        fs1 = fin.getframerate()
-        audio1 = np.frombuffer(fin.readframes(fin.getnframes()), np.int16)
-
-    with wave.open(args.audio2, 'rb') as fin:
-        fs2 = fin.getframerate()
-        audio2 = np.frombuffer(fin.readframes(fin.getnframes()), np.int16)
+    fin = wave.open(args.audio1, 'rb')
```
Comment (lissyx) [inline native_client/test/concurrent_streams.py]: Why not using `with` ?

Comment (reuben) [reply native_client/test/concurrent_streams.py]: Python 2.7 wave.open objects don't support that syntax.

**Pull request merged**